Lib/test/test_hash.py

```diff
@@ -206,18 +206,18 @@ class StringlikeHashRandomizationTests(HashRandomizationTests):
             # seed 42, 'abc'
             [-678966196, 573763426263223372, -820489388, -4282905804826039665],
             ],
-        'siphash24': [
+        'siphash': [
```

Comment:
`test_hash` aborts with `KeyError: 'siphash13'`

```suggestion
        'siphash13': [
```

Alternatively, change `test_hash` to look for `siphash` instead. (Probably a better solution.)